docs/eventing/README.md

```diff
@@ -61,3 +61,7 @@ the registry contain (all) the required information for a consumer to create a
 Trigger without resorting to some other out-of-band mechanism.
 
 To learn how to use the registry, see the [Event Registry documentation](event-registry.md).
+
+### Installation
```

Comment:
I agree with @psschwei that these should be H2 headings. I just took a look at the [preview](https://deploy-preview-4853--knative.netlify.app/development/eventing/), and these new sections are quite hidden on the page. 

Also consider moving this further up because the intention of this issue was to help users see installation information more easily.